faq.md

```diff
@@ -12,18 +12,13 @@ permalink: /faq/
 
 Yes, <a href="download/">choose a download</a>.
 
-Don't be an idiot and download a binary claiming to be of this project. In fact,
-don't be an idiot and download *any* binary claiming to be an Xbox 360 or PS3
-emulator from *any* source, especially not YouTube videos and shady websites.
-Come on people. Jeez.
-
 ## Xenia crashes when I run it!
 
+Make sure your PC meets the system requirements of xenia.
```

Comment:
Xenia should be capitalized, and "system requirements" could be a link to the section in the FAQ.